### DISCUSSION TITLE
Need Dabbaba movement.

### DISCUSSION THREAD

**Discussion by LiuCheVN** [fairy-stockfish/Fairy-Stockfish#510] [General]
(above)

**Comment by ianfab** [comment]
See https://github.com/ianfab/Fairy-Stockfish/blob/7561e85219f8b1ec0dc09f921f3fdcabd3a4a173/src/variants.ini#L87-L107 and the link https://www.gnu.org/software/xboard/Betza.html mentioned there.

**Reply by LiuCheVN** [reply]
And the thing is: you can't make any fairy pieces move like Dabbaba (or a part of that) using Fairy-Stockfish's Betza notation subset.

**Reply by ianfab** [reply]
No, it does, just try. And paste your config here if it still doesn't work then.

**Reply by LiuCheVN** [reply]
Ok.
It works!!!!!
(YAYYYYYYYYYYYYYYYYYYYYYYYY!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!)
